examples/module-federation/README.md

```diff
@@ -1,17 +1,18 @@
 # Module Federation
 
-**TLDR** : Run the following commands in the root directory.
+## How to use
 
-```
-yarn install
+Run the following commands in the root directory.
 
-yarn start (this should trigger the lerna command)
+```bash
+❯ npm install
+❯ npm start
```

Comment:
```suggestion
$ npm install
$ npm start
```

Reply:
Another thing I found is this example won't work under npm 6 in my test. Only yarn works.

Reply:
Oh interesting. That's on local then right?

Reply:
Right, I'm testing on my local computer.